core/src/OpenACC/Kokkos_OpenACC_Instance.hpp

```diff
@@ -35,15 +35,15 @@ class OpenACCInternal {
 
  public:
   static int m_acc_device_num;
-  int m_async_arg = acc_async_sync;
+  int m_async_arg = acc_async_noval;
```

Comment:
What's the difference between `acc_async_noval` and `acc_async_sync`?

Reply:
`acc_async_novel` means that the kernels are submitted to a default asynchronous queue, and `acc_async_sync` means that kernels are submitted to a default synchronous queue.